packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -70,10 +92,23 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
             [`& .${getDataGridUtilityClass("booleanCell")}`]: {
                 color: palette.grey[900],
             },
-        },
-        footerContainer: {
+            height: `${getDensityHeightValue(ownerState?.density, spacing)}`,
+            alignContent: "center",
+        }),
+        footerContainer: ({ ownerState }) => ({
             borderTop: `1px solid ${palette.grey[100]}`,
-        },
+            boxSizing: "border-box",
+            /* !important is required to override inline styles */
+            height: `${getDensityHeightValue(ownerState?.density, spacing)} !important`,
```

Comment:
There does not seem to be an existing `height` value, so this should not be necessary. 

```suggestion
```